targets/TARGET_STM/TARGET_STM32F1/TARGET_BLUEPILL_F103C8/device/stm32f103xb.h

```diff
@@ -10630,6 +10630,24 @@ typedef struct
   * @}
   */
 
+#define SC_USART                           USART3
```

Comment:
I would prefer that this change coming from MSP layer do not end in device/ directory.
Those files are application layer dependent as other application may want other pins than PB_10 / PB_11. So I think that you should rather add them to your project only